apps/dot_ext/migrations/0010_auto_20181219_2117.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 2.1.2 on 2018-12-19 21:17
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dot_ext', '0009_auto_20181112_1856'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='application',
+            name='client_uri',
+        ),
+        migrations.AddField(
+            model_name='application',
+            name='website_uri',
+            field=models.URLField(blank=True, default='', help_text='This is typically a home/download website for the application.', max_length=512, verbose_name='Website URI'),
+        ),
+        migrations.AlterField(
```

Comment:
remove these migrations, only because they're unrelated / expected given the rest of the change set.

Reply:
Removed the unrelated AlterField operations.